torch/csrc/CudaIPCTypes.h

```diff
@@ -0,0 +1,162 @@
+#pragma once
+#ifdef USE_CUDA
+#include <c10/core/Allocator.h>
+#include <c10/cuda/CUDACachingAllocator.h>
+#include <c10/cuda/CUDAException.h>
+#include <c10/cuda/CUDAGuard.h>
+#include <c10/cuda/CUDAStream.h>
+#include <c10/util/Logging.h>
+#include <cuda_runtime_api.h>
+#include <iostream>
+
+namespace torch {
```

Comment:
A little nit: in situation like this, where the functions and structs are internal usage and not to be available to users from the public `torch::` API, consider putting them in a sub-namespace. This is especially helpful if someone is trying to use autocomplete on `torch::`; avoiding random stuff makes the autocomplete more useful.

Reply:
done